src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -220,9 +223,9 @@ def step(
         model_output: torch.FloatTensor,
         timestep: int,
         sample: torch.FloatTensor,
-        predict_epsilon=True,
```

Comment:
`predict_epsilon` is an inherent config parameter just like `beta_start` that should not change during inference